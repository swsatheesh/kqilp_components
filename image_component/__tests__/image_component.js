import React from 'react';
import renderer from 'react-test-renderer';
import ImageComponent from '../image_component';

// Mock store setup.

jest.unmock('../image_component');

describe('components', () => {
  describe('<ImageComponent />', () => {
    it('should render ImageComponent', () => {
      const comp = renderer.create(
        <ImageComponent
          src={test.png}
          alt={''}
        />
      ).toJSON();
      expect(comp).toMatchSnapshot();
    });
  });
});
