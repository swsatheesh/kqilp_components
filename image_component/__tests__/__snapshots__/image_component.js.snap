// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <ImageComponent /> should render ImageComponent 1`] = `
<img
  alt=""
  role="presentation"
  src="default.png"
/>
`;
